### ISSUE TITLE
Generating Data Matrix code

### ISSUE THREAD

**Issue by kungfupizza** [ricmoo/QRCode#14]
Hi,
Is there a way to generate a Data Matrix code from this library?
Like this one in Image.

![image](https://user-images.githubusercontent.com/46355685/68842799-4854d880-06c7-11ea-8387-a82b69440ccf.png)

**Comment by ricmoo** [comment]
No, this library is not compatible with data matrixes. Sorry. :s
